ios/AirMaps/AIRMap.m

```diff
@@ -67,6 +73,23 @@ - (instancetype)init
         // be identical to the built-in callout view (which has a private API)
         self.calloutView = [SMCalloutView platformCalloutView];
         self.calloutView.delegate = self;
+        
+        self.cacheImageView = [[UIImageView alloc] initWithFrame:self.bounds];
+        self.cacheImageView.autoresizingMask = UIViewAutoresizingFlexibleWidth|UIViewAutoresizingFlexibleHeight;
+        [self addSubview:self.cacheImageView];
+        self.cacheImageView.hidden = YES;
+        
+        self.loadingView = [[UIView alloc] initWithFrame:self.bounds];
+        self.loadingView.autoresizingMask = UIViewAutoresizingFlexibleWidth|UIViewAutoresizingFlexibleHeight;
+        self.loadingView.backgroundColor = [UIColor whiteColor];
+        [self addSubview:self.loadingView];
+        self.loadingView.hidden = NO;
+        
+        self.activityIndicatorView = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle:UIActivityIndicatorViewStyleGray];
+        [self.loadingView addSubview:self.activityIndicatorView];
+        self.activityIndicatorView.center = self.loadingView.center;
+        self.activityIndicatorView.autoresizingMask = UIViewAutoresizingFlexibleTopMargin | UIViewAutoresizingFlexibleBottomMargin | UIViewAutoresizingFlexibleLeftMargin | UIViewAutoresizingFlexibleRightMargin;
+        [self.activityIndicatorView startAnimating];
```

Comment:
if the loadingEnabled prop is false by default, do we want to start the animation in every case? or make the view visible?  

I'm thinking for this feature we would want to add this capability without changing the previous behavior so that it's an enhancement rather than a "breaking" change / difference.